trick-2.php

```diff
@@ -1,6 +1,18 @@
 <?php
 
-/* Insert Magic Code Here */
+$array = new class implements Countable {
```

Comment:
wow. that's a nice solution!
my way was just to instantiate the array with a single reference: `$array = [&$var, &$var, &$var];`